src/components/Card/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export interface LinkProps {
+  label?: string;
+  url?: string;
+}
+
+const Link: React.FC<LinkProps> = ({ label, url }: LinkProps) => {
+  return (
+    <a
+      href={url}
+      className="text-lg no-underline text-blue-600 hover:text-blue-800 visited:text-purple-600"
+    >
+      {label}
+    </a>
+  );
+};
+export interface CardProps extends LinkProps {
+  title: string;
+  description: string;
+  isLink: boolean;
+}
+
+const Card: React.FC<CardProps> = ({
+  title,
+  description,
+  isLink,
+  url,
+  label,
+}: CardProps) => {
+  return (
+    <div
+      data-testid="card-element"
+      className="max-w-md py-4 px-8 bg-white shadow-lg rounded-lg my-20"
+    >
+      <div className="text-gray-800 text-3xl font-semibold">{title}</div>
```

Comment:
I think that `title` could be optional. Sometimes we don't want to have it.

Reply:
Made all the card properties optional.